treeherder/intermittents_commenter/commenter.py

```diff
@@ -447,7 +483,10 @@ def build_bug_map(self, bugs, option_collection_map):
                 bug_infos.total = 1
                 bug_infos.test_variants |= all_variants
                 bug_infos.per_repositories[repo] = 1
-                bug_infos.data_table[platform_and_build] = {test_variant: 1}
+                bug_infos.data_table[platform_and_build] = {
+                    test_variant: 1,
+                    "total_runs": run_count,
+                }
```

Comment:
maybe to support the variants, it could be:
`bug_infos.data_table[platform_and_build] = {'test_variant': {'count': 1, 'runs': run_count}}`